HHS/TANF-app
# 1336 access request email

## Summary of Changes
- Added new task to send OFA System Admins an email every day with the number of access requests
- Added new field to user model to track when user requested access
- Updated admin user view to display when access was requested/make it sortable

## How to Test
- You'll want to update the frequency of the cron job in `settings/common.py` to see the email printed out to the console more frequently
```
git checkout 1336-access-request-email
cd tdrs-frontend && docker-compose
cd tdrs-backend && docker-compose
```

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [ ] `OFA System Admin` receives one email per day (ideal run time between 0100-0200 UTC) presenting a count of all users with an "Access Request" status as of that day. Prototype in notes.
+ [ ] Django Admin console has additional filter to show timestamp of when a user's status changed to, "Access request"

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
